src/main/java/net/sf/jabref/logic/l10n/Encodings.java

```diff
@@ -9,12 +9,12 @@
 public class Encodings {
 
     public static final String[] ALL_ENCODINGS =
-            new String[] {"ISO8859_1", "UTF8", "UTF-16", "ASCII", "Cp1250", "Cp1251", "Cp1252",
-                    "Cp1253", "Cp1254", "Cp1257", "SJIS",
-                    "KOI8_R", // Cyrillic
-                    "EUC_JP", // Added Japanese encodings
-                    "Big5", "Big5_HKSCS", "GBK", "ISO8859_2", "ISO8859_3", "ISO8859_4", "ISO8859_5",
-                    "ISO8859_6", "ISO8859_7", "ISO8859_8", "ISO8859_9", "ISO8859_13", "ISO8859_15"};
+            new String[] {"ISO8859-1", "UTF-8", "UTF-16", "ASCII", "CP1250", "CP1251", "CP1252",
+                    "CP1253", "CP1254", "CP1257", "SJIS",
+                    "KOI8-R", // Cyrillic
+                    "EUC-JP", // Added Japanese encodings
+                    "Big5", "Big5-HKSCS", "GBK", "ISO8859-2", "ISO8859-3", "ISO8859-4", "ISO8859-5",
+                    "ISO8859-6", "ISO8859-7", "ISO8859-8", "ISO8859-9", "ISO8859-13", "ISO8859-15"};
```

Comment:
Yes, this requires major changes, since all of the code relies on strings. I'd rather do this in a second issue / PR.